src/guest/vm_builder_qemu.cc

```diff
@@ -68,6 +68,8 @@ constexpr const unsigned int kExecQuantum = 25;
 
 constexpr const char *kQmpPowerSocket = "/tmp/qmp-pwr-socket-";
 
+constexpr const char *kIntelBtPciAddress = "0000:00:14.0";
```

Comment:
I think `00:14.0` is not always IntelBT device, right? There should be another way to detect bluetooth device such like check vendor id or device id.